### DISCUSSION TITLE
== Error ==  system call for: finished abnormally, OS return value: 4

### DISCUSSION THREAD

**Discussion by sophieleech** [ablab/spades#977] [Q&A]
Hi, 

I'm running metaspades and it has successfully run on most of my files, however 7 or so of them are returning errors similar to this. 

== Error ==  system call for: "['/Users/uqsleech_local/opt/anaconda3/envs/biobakery3/bin/spades-hammer', '/Users/uqsleech_local/metaSPAdes_output/113a/corrected/configs/config.info']" finished abnormally, OS return value: 4
None

Here are my spades.log and params.txt files from this run: 
[spades.log](https://github.com/ablab/spades/files/8775384/spades.log)
[params.txt](https://github.com/ablab/spades/files/8775385/params.txt)

Thank you!

**Comment by asl** [comment]
The log reads:

```
Exception caught Cannot create temporary dir /Users/uqsleech_local/metaSPAdes_output/113a/tmp/hammer_9qj_o91q/kmer_splitter_XXXXXX
```

So, it looks like an i/o problem on your end

**Reply by sophieleech** [reply]
Hi,

Thanks so much for the response. Do you have any idea why this would only be occurring for certain files? I tried rerunning files that had previously completed correctly and they ran fine, but these particularly files continue to have issues though are now giving a slightly different error. 

Assertion failed: (is.good()), function process, file /Users/asl/Temp/SPAdes-3.15.4/src/projects/hammer/kmer_cluster.cpp, line 621.

== Error ==  system call for: "['/Users/uqsleech_local/opt/anaconda3/envs/biobakery3/bin/SPAdes-3.15.4-Darwin/bin/spades-hammer', '/Users/uqsleech_local/metaSPAdes_output/081a/corrected/configs/config.info']" finished abnormally, OS return value: -6
None

[spades.log](https://github.com/ablab/spades/files/8794432/spades.log)

[params.txt](https://github.com/ablab/spades/files/8794433/params.txt)

**Reply by asl** [reply]
You may want to check your system log. E.g. you might be out of free disk space, or your disk quota might be exhausted, etc.

**Reply by Cyberia1978** [reply]
Hi Sophy, have you solved the problem? I have the same and I don't know how to fix it...:(
